tests/test_driver.py

```diff
@@ -6,8 +6,8 @@
 from pytest import mark, raises
 from requests.utils import default_headers
 from sha3 import sha3_256
-
 from cryptoconditions import Ed25519Sha256
+import asyncio
```

Comment:
I see the only function used from the `asyncio` module is `asyncio.sleep`. Why not using `time.sleep`? It would simplify the tests.

Reply:
the main reason why i included ```asyncio```, is to make use of its features on future multiple nodes tests and the following ```FIXME``` comment in conftest.py, line 16: 
```(python)
# FIXME The sleep, or some other approach is required to wait for the
# transaction to be available as some processing is being done by the
# server.
```
let me know if it makes sense, otherwise i'll remove it.

Reply:
You do not need to sleep at all. Transactions are now posted [in the commit mode](https://github.com/bigchaindb/bigchaindb-driver/blob/master/tests/conftest.py#L193) so when you get the response the data is stored and available by definition.

The `await_transaction` fixture (you referred to its FIXME) is a confusing piece of legacy code, that is going to be removed soon.